tasks/main.yml

```diff
@@ -24,6 +24,53 @@
   when:
     - network_state is defined
     - ansible_distribution_major_version | int < 8
+
+- name: Check if updates for network packages are available through the DNF
+    package manager due to wireless or team interfaces
+  dnf:
```

Comment:
Please explain in the commit message why we have both dnf and yum tasks.

Reply:
I explained it.